res/graphics/authors.txt

```diff
@@ -322,6 +322,7 @@ quest/
     fire_hydrant.svg
     fire_hydrant_diameter.svg
     fire_hydrant_grass.svg
+    fire_hydrant_ref.svg             modified from fire_hydrant.svg by mcliquid
```

Comment:
The "REF" letters are too small for the icon. I suggest you either overlay the hydrant picture with the "label" icon (at position of current REF but a little large) or with the "ID" from the mailbox ref quest icon (centered and rather on top)